src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -403,8 +403,8 @@ class AdminApiServiceSpec extends ApiServiceSpec {
     Get("/admin/users") ~>
       sealRoute(services.adminRoutes) ~>
       check {
-        assertResult(RawlsUserInfoList(Seq(userOwner, userWriter, userReader)), response) {
-          responseAs[RawlsUserInfoList]
+        assertResult(Set(userOwner, userWriter, userReader), response) {
```

Comment:
The order of the results changed.  I don't know why, but the order shouldn't matter, so...